build.sbt

```diff
@@ -43,6 +43,11 @@ def versionedScalaJvmSourceDirectories(sourceDir: File, scalaVersion: String): L
     case _            => List(sourceDir / "scalajvm-2")
   }
 
+def inlineOptions(scalaVersion: String) = CrossVersion.partialVersion(scalaVersion) match {
+    case Some((3, _)) => Seq("-Xmax-inlines", "64")
```

Comment:
oh, this change increased the inlining requirements?

Reply:
reverted that change